README.md

```diff
@@ -45,23 +45,74 @@ The following procedure describes how to deploy the `ExternalDNS` Operator for A
 
 
 ### Installing the `ExternalDNS` Operator using a custom index image on OperatorHub
+#### Note By default container engine used is docker but you can specify podman by adding CONTAINER_ENGINE=podman to your image build and push commands as mentioned below.
+    
 
-1. Build and push the bundle image to a registry:
+1. Build the bundle image to a registry:
+  
+  a. Export your expected image name which shall have your registry name in an env var.
    ```sh
    $ export BUNDLE_IMG=<registry>/<username>/external-dns-operator-bundle:latest
-   $ make bundle-image-build
-   $ make bundle-image-push
+   ```
+   
+  b. In the bundle/manifests/external-dns-operator_clusterserviceversion.yaml
+   ```sh
+    In annotations:
+    Change containerImage: quay.io/openshift/origin-external-dns-operator:latest
+    to containerImage: <registry>/<username, repository name>/external-dns-operator:latest
+   
+    In spec:
+    Change image: quay.io/openshift/origin-external-dns-operator:latest
+    to image: <registry>/<username, repository name>/external-dns-operator:latest
+  ```
+  
+  c. Build the image
+   ```sh   
+   $ CONTAINER_ENGINE=<podman/docker> make bundle-image-build
```

Comment:
I think that specifying `CONTAINER_ENGINE` for all the commands is superfluous as:
- you have already made a note about it at the beginning
- `CONTAINER_ENGINE` can be specified once by the person who follows the guide